raiden/raiden_service.py

```diff
@@ -1,73 +1,134 @@
+)
+from raiden.transfer.architecture import (
+    StateManager,
+)
+from raiden.transfer.state_change import (
+    Block,
+)
+from raiden.transfer.state import (
+    RoutesState,
+)
+from raiden.transfer.mediated_transfer import (
+    initiator,
+    mediator,
+)
+from raiden.transfer.mediated_transfer import target as target_task
+from raiden.transfer.mediated_transfer.state import (
+    lockedtransfer_from_message,
+    InitiatorState,
+    MediatorState,
+    LockedTransferState,
+)
+from raiden.transfer.mediated_transfer.state_change import (
+    ActionInitInitiator,
+    ActionInitMediator,
+    ActionInitTarget,
+    ContractReceiveBalance,
+    ContractReceiveClosed,
+    ContractReceiveNewChannel,
+    ContractReceiveSettled,
+    ContractReceiveTokenAdded,
+    ContractReceiveWithdraw,
+    ReceiveSecretRequest,
+    ReceiveSecretReveal,
+    ReceiveTransferRefund,
+)
+from raiden.transfer.mediated_transfer.events import (
+    EventTransferCompleted,
+    EventTransferFailed,
+    SendBalanceProof,
+    SendMediatedTransfer,
+    SendRefundTransfer,
+    SendRevealSecret,
+    SendSecretRequest,
+)
+from raiden.channel import ChannelEndState, ChannelExternalState
+from raiden.exceptions import (
     UnknownAddress,
-    UnknownTokenAddress
+    TransferWhenClosed,
+    UnknownTokenAddress,
+    NoPathError,
+    InvalidAddress,
+    InvalidAmount,
+    InvalidState,
+    InsufficientFunds,
 )
-from raiden.blockchain.abi import (
-    CHANNEL_MANAGER_ABI,
-    REGISTRY_ABI,
-    NETTING_CHANNEL_ABI
+from raiden.network.channelgraph import (
+    channel_to_routestate,
+    route_to_routestate,
+    ChannelGraph,
+    ChannelDetails,
 )
-from raiden.network.channelgraph import ChannelGraph
-from raiden.tasks import AlarmTask, StartExchangeTask, HealthcheckTask
 from raiden.encoding import messages
-from raiden.messages import SignedMessage
+from raiden.messages import (
+    RevealSecret,
+    Secret,
+    SecretRequest,
+    SignedMessage,
+)
 from raiden.network.protocol import RaidenProtocol
 from raiden.utils import (
     isaddress,
     pex,
     privatekey_to_address,
     safe_address_decode,
+    sha3,
     GLOBAL_CTX,
 )
 
-
 log = slogging.get_logger(__name__)  # pylint: disable=invalid-name
 
 
-EventListener = namedtuple('EventListener', ('event_name', 'filter_', 'translator'))
-
-
-class RaidenError(Exception):
-    pass
-
-
-class NoPathError(RaidenError):
-    pass
-
-
-class InvalidAddress(RaidenError):
-    pass
-
-
-class InvalidAmount(RaidenError):
-    pass
-
+def create_default_identifier(node_address, token_address, target):
+    """
+    The default message identifier value is the first 8 bytes of the sha3 of:
+        - Our Address
+        - Our target address
+        - The token address
+        - A random 8 byte number for uniqueness
+    """
+    hash_ = sha3('{}{}{}{}'.format(
+        node_address,
+        target,
+        token_address,
+        random.randint(0, 18446744073709551614L)
```

Comment:
shouldn't this be `2 ** 64 - 1`? 
also consider using a constant for the max value `MAX_I64 = 2 ** 64 - 1`